tokens.xml

```diff
@@ -3931,8 +4016,8 @@ When this creature enters the battlefield, it deals 1 damage to any target.</tex
             <reverse-related>Battle for Bretagard</reverse-related>
             <reverse-related>Beskir Shieldmate</reverse-related>
             <reverse-related>Maja, Bretagard Protector</reverse-related>
-            <reverse-related exclude="exclude" count="x">Maja, Bretagard Protector</reverse-related>
-            <reverse-related count="x=1">Usher of the Fallen</reverse-related>
+            <reverse-related count="x" exclude="exclude">Maja, Bretagard Protector</reverse-related>
+            <reverse-related>Usher of the Fallen</reverse-related>
```

Comment:
@Psithief added the `count="x=1"` in https://github.com/Cockatrice/Magic-Token/commit/f0b7fee4eb1ccd9dbf2f23a0d9b53acd7ed173d1#diff-ef2c5b2e0216513ba7314de2c6637a3b48b09a5ea3ee44d13c951195fb293697R3716

Maybe the idea was to account for something like double Boast triggers?

But looking at other Boast cards that create tokens they also have no count value. 👍 
https://scryfall.com/search?q=o%3Aboast+o%3Atoken

Reply:
I think the count for this relation should be removed as Boast is once per turn, and none of the other token relations assume a token doubler or similar.

Reply:
>I think the count for this relation should be removed as Boast is once per turn, and none of the other token relations assume a token doubler or similar.

Agreed.